### ISSUE TITLE
New version isn't loading skada

### ISSUE THREAD

**Issue by wojas007** [bkader/Skada-Revisited#9]
Hello,
Today my friend have sent me link to your revisited Skada. I was pretty hyped and downloaded it but it for some reason I couldn't get it to work (was not loading into the game despite showing in addon selection on logging screen). My friend gave me 1 week old version and it is working fine (he could not load newest version too). No idea what is the problem.

Regards,
Wojas

**Comment by bkader** [comment]
Have you deleted all **Skada** folders and modules (default Skada), installed this new one, deleted old **SavedVariables** and tried? Because it is working just fine for me! @wojas007

**Comment by wojas007** [comment]
Let me rephrase what I wrote above. I have 2 versions of YOUR skada. One is the newest (downloaded from here yesterday) and other one is older version of YOUR revised Skada from a friend who is using your addon.

What I did yesterday (followed instructions from installation):
- download skada from here
- WoW\Interface\Addons -> delete all Skada (and SkadaXXX related folders - modules)
- extract your skada into Addons (folder named "Skada" is now in the Addons)
- WTF\Account\[accname]\SavedVariables -> delete everything that starts with Skada (I did that for every account I have)
- Run WoW -> Skada shows in logging screen in addon selection but nothing loads into the game. I tried typing /skada or checking Interface\Addons tab in the game but nothing was there.

Then my friend (who is using YOUR revised Skada) gave me his version of YOUR addon (like 1 week old). I followed the steps again and it is working properly now (https://i.ibb.co/VN83wvM/image.png).

Today after your message I tried to replace the Addon version for the newest from here (without deleting settings), but it still does not load for me).

**Comment by bkader** [comment]
This Skada stores data within accout SavedVariables amd characters SavedVariables, this is why I said "search for all files named Skada inside WTF folder and delete them", can you try this and let me know please.

**Comment by wojas007** [comment]
Well I did that. Only files that were found with "Skada" were in account SavedVariables, but I went to every character\SavedVariables and checked wheter there is any Skada file or not. To cut the long story short: I followed the instructions exactly as it is written.

If this was a problem with my actions, then none of your skada versions would work for me. But as I said before the version from a week ago is working fine. I only have problem with the newest version.

But don't worry about me... Let's wait until someone else runs into this problem...

**Comment by wojas007** [comment]
https://i.ibb.co/nLSxf0t/image.png

**Comment by bkader** [comment]
> 
> 
> https://i.ibb.co/nLSxf0t/image.png

Please delete the Skada you have, download the latest version and re-install it. You're missing files.

**Comment by wojas007** [comment]
What the fuck now it is working... I literally downloaded it yesterday from here... I am really sorry for the trouble...

**Comment by bkader** [comment]
No worries at all, glad it's working now. Please feel free to report any bugs you find and all your suggestions are welcome as long as they remain in the addon concept.
